docs/en/sql-reference/window-functions/lead.md

```diff
@@ -0,0 +1,71 @@
+---
+description: 'Documentation for the lead window function'
+sidebar_label: 'lead'
+sidebar_position: 10
+slug: /sql-reference/window-functions/lead
+title: 'lead'
+---
+
+# lead
+
+Returns a value evaluated at the row that is offset rows after the current row within the ordered frame.
+This function is similar to [`leadInFrame`](./leadInFrame.md), but always uses the `ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING` frame.
+
+**Syntax**
+
+```sql
+lead(x[, offset[, default]])
+  OVER ([[PARTITION BY grouping_column] [ORDER BY sorting_column]] | [window_name])
+FROM table_name
+WINDOW window_name as ([[PARTITION BY grouping_column] [ORDER BY sorting_column])
+```
+
+For more detail on window function syntax see: [Window Functions - Syntax](./index.md/#syntax).
+
+**Parameters**
+
+- `x` — Column name.
+- `offset` — Offset to apply. [(U)Int*](../data-types/int-uint.md). (Optional - `1` by default).
+- `default` — Value to return if calculated row exceeds the boundaries of the window frame. (Optional - default value of column type when omitted).
+
+**Returned value**
+
+- value evaluated at the row that is offset rows after the current row within the ordered frame.
+
+**Example**
+
+This example looks at [historical data](https://www.kaggle.com/datasets/sazidthe1/nobel-prize-data) for Nobel Prize winners and uses the `lead` function to return a list of successive winners in the physics category.
+
+```sql title="Query"
+CREATE OR REPLACE VIEW nobel_prize_laureates
+AS SELECT *
+FROM file('nobel_laureates_data.csv');
+```
+
+```sql title="Query"
+SELECT
+    fullName,
+    lead(year, 1, year) OVER (PARTITION BY category ORDER BY year ASC
+      ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING
+    ) AS year,
+    category,
+    motivation
+FROM nobel_prize_laureates
+WHERE category = 'physics'
+ORDER BY year DESC
+LIMIT 9
```

Comment:
Maybe I'm missing something, but isn't this example totally wrong/breaking a dataset?

This is pulling the year from the next row in the same category, which might not even be the same person. This is just shifting year that a prize in physics was awarded to the next year a prize was awarded, and has no relation to the current row other than being 1 offset within the category...

TLDR it's just mangling the dataset AFAICT, right? I did consult with o4-mini-high to make sure that I wasn't interpreting it wrong, but it seems like were just shifting the year and mangling the data for no reason. I'm not sure this is a useful example.

Reply:
@danthegoodman1 I think you are right. I will redo these examples.